app/subscriber/src/features/my-folders/styled/ConfigureFolder.tsx

```diff
@@ -53,9 +53,31 @@ export const ConfigureFolder = styled(PageSection)`
       filter: grayscale(0.2);
     }
   }
+  .react-datepicker__time-list-item--selected,
   .react-datepicker__close-icon::after {
+    background-color: rgb(151, 29, 41) !important;
+  }
+  .react-datepicker__close-icon::after {n
     background-color: ${(props) => props.theme.css.btnBkPrimary};
   }
+  .react-datepicker__day--keyboard-selected,
+  .react-datepicker__day--in-selecting-range {
+    background-color: rgb(151, 29, 41, 0.5);
+  }
+
+  .react-datepicker__day--in-range,
+  .react-datepicker__day--selected {
+    background-color: rgb(151, 29, 41);
+  }
+
+  .react-datepicker__day--keyboard-selected,
+  .react-datepicker__month-text--keyboard-selected,
+  .react-datepicker__quarter-text--keyboard-selected,
+  .react-datepicker__year-text--keyboard-selected {
+    &:hover {
+      background-color: rgb(151, 29, 41);
```

Comment:
Might be worth adding these colors to our variables - other than that looks good!